package.json

```diff
@@ -33,6 +33,7 @@
     "@types/react-hot-loader": "^4.1.0",
     "@types/react-router": "^4.4.1",
     "@types/react-router-dom": "^4.3.1",
+    "@types/stack-trace": "0.0.29",
```

Comment:
i _think_ this will solve why https://github.com/HedvigInsurance/web-next/pull/18 is failing. nice!